bind_uploader.go

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) 2021 Snowflake Computing Inc. All right reserved.
+
+package gosnowflake
+
+//lint:file-ignore U1000 Ignore all unused code
+
+import (
+	"bytes"
+	"context"
+	"database/sql/driver"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+const (
+	stageName       = "SYSTEM$BIND"
+	createStageStmt = "CREATE TEMPORARY STAGE " + stageName + " file_format=" +
+		"(type=csv field_optionally_enclosed_by='\"')"
+
+	inputStreamBufferSize      = 1024 * 1024 * 10
```

Comment:
Suggest adding comments for this. How did we come up with this any references etc.